src/vs/workbench/services/hover/browser/hoverWidget.ts

```diff
@@ -20,6 +20,8 @@ import { MarkdownRenderer, openLinkFromMarkdown } from 'vs/editor/contrib/markdo
 import { isMarkdownString } from 'vs/base/common/htmlContent';
 import { localize } from 'vs/nls';
 import { isMacintosh } from 'vs/base/common/platform';
+import { IAccessibilityService } from 'vs/platform/accessibility/common/accessibility';
+import { getHoverAriaLabel } from 'vs/editor/contrib/hover/browser/contentHover';
```

Comment:
Can this be moved to `base/browser/ui/hover`? I know it uses services but I don't want the workbench hover to depend on editor hover at all, we can pass in values from the services into getHoverAriaLabel in base

Reply:
not much benefit to having a shared function if everything is passed in

Reply:
so i'll just put it in both places